### DISCUSSION TITLE
increase indent for bullet list in latex

### DISCUSSION THREAD

**Discussion by blaplafla13th** [jgm/pandoc#10193] [Q&A] [answered]
I want to make this bullet list get more indent or have bigger left margin because paragraph before it have only 1 line and i using package `indentfirst`. How to do it?

![received_1546429245999822.jpeg](https://github.com/user-attachments/assets/e6c4057b-d255-4810-ba9f-9f2e2d9f8e08)

**Comment by jgm** [comment]
Check out the latex package `enumitem`, which allows you to set a leftmargin for any type of list using `\setlist`.

**Reply by blaplafla13th** [reply]
Thanks, but i don't want setting it globally

**Comment by jgm** [comment, marked as answer]
You could embed the list in a block quote?
